server/views/partials/layout.njk

```diff
@@ -63,19 +63,31 @@
       }
     }) }}
 
-    {% if successMessage %}
```

Comment:
While testing I noticed I put this in the wrong place in the other PR so have moved it as well as adding the error message banner.